graphbolt/src/cnumpy.cc

```diff
@@ -21,6 +21,7 @@ OnDiskNpyArray::OnDiskNpyArray(
     std::string filename, torch::ScalarType dtype, torch::Tensor shape)
     : filename_(filename), dtype_(dtype) {
 #ifdef __linux__
+#ifdef HAVE_LIBRARY_LIBURING
```

Comment:
why not merge `__linux__` and `HAVE_LIBRARY_LIBURING` into single one such as `__liburing__` which is defined only when system is linux and `USE_LIBURING` is `ON`?